client/src/layout/menu.js

```diff
@@ -114,38 +114,49 @@ export function fetchMenu(options = {}) {
     //
     // 'Shared Items' or Libraries tab.
     //
-    menu.push({
-        id: "shared",
-        title: _l("Shared Data"),
-        url: "javascript:void(0)",
-        tooltip: _l("Access published resources"),
-        menu: [
-            {
-                title: _l("Data Libraries"),
-                url: "libraries",
-            },
-            {
-                title: _l("Histories"),
-                url: "histories/list_published",
-                target: "__use_router__",
-            },
-            {
-                title: _l("Workflows"),
-                url: "workflows/list_published",
-                target: "__use_router__",
-            },
-            {
-                title: _l("Visualizations"),
-                url: "visualizations/list_published",
-                target: "__use_router__",
-            },
-            {
-                title: _l("Pages"),
-                url: "pages/list_published",
-                target: "__use_router__",
-            },
-        ],
-    });
+    if (Galaxy.config.single_user) {
+        // Single user can still use libraries, especially as we may grow that
+        // functionality as a representation for external data.  The rest is
+        // hidden though.
+        menu.push({
+            title: _l("Data Libraries"),
+            url: "library/list",
```

Comment:
here's the problem, it should be libraries

```suggestion
            url: "libraries",
```

Reply:
Awesome, thank you!